alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/fixer/solr/AbstractSolrNodeFixerPlugin.java

```diff
@@ -41,6 +41,8 @@
     protected abstract Set<NodeFixReport> handleHealthReport(NodeHealthReport unhealthyReport,
             NodeIndexHealthReport endpointHealthReport);
 
+    protected abstract void clearCache();
```

Comment:
I am not too happy with the introduction of this vague method name that is only useful in one child class.

Since it's called at the beginning of the `fix` method, can't we move it to the child class, override `fix` and call `super.fix` before/after clearing the "cache"?